ui/apps/platform/src/messages/vulnMgmt.messages.ts

```diff
@@ -48,4 +48,8 @@ export const nodeScanMessages = {
         header: 'Node’s kernel unsupported.',
         body: 'This node’s kernel is not supported at this time.',
     },
+    certifiedRHELCVEsUnavailable: {
+        header: 'Unavailable or invalid CPE data',
```

Comment:
```suggestion
        header: 'Unable to scan node components',
```

Let's refrain from using the CPE abbreviation in customer facing communication.

Reply:
Okay